fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt

```diff
@@ -95,6 +96,7 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         persistentSheet = findViewInTemplateById(R.id.persistent_bottom_sheet)!!
         persistentSheetContainer = findViewInTemplateById(R.id.persistent_sheet_container)!!
         persistentSheetBehavior = BottomSheetBehavior.from(persistentSheet)
+        persistentSheetBehavior.isHideable = true
```

Comment:
just for my info.
Why do we need this now? and why it was not used earlier?

Reply:
earlier code was hiding using peek height animation which is not very scalable,(like mutiple calls from an event listener etc)
this flag is required for settings state hidden